docker-compose.yml

```diff
@@ -43,6 +43,70 @@ services:
     networks:
       - primenet
 
+####################### Verifiable Credential Agent Definition #######################
```

Comment:
This is commented out on purpose because the agent and wallet cannot be started using simply docker-compose up. it must have its environment variables set using the manage script (and should be uncommented when doing vc dev or testing only).